website/docs/components/drag-and-drop/drag-and-drop-a11y.md

```diff
@@ -11,50 +11,64 @@ tabs: Design('drag-and-drop'), A11y('drag-and-drop-a11y'), API('drag-and-drop-ap
+| --------------------------- | -------------------------------------------------------------------------- |
+| User grabbed the object     | `{itemText} grabbed. Current position: {itemPosition} out of {itemsCount}. Use the arrows to change the position, Space to apply the new position, Escape to cancel.` |
+| User moved the object       | `New position: {itemPosition} out of {itemsCount}`                         |
+| User dropped the object     | `Position {itemPosition} applied`                                          |
+| User canceled drag and drop | `Drag and drop canceled`                                                   |
 
-If grabbed item or zone to drop has `zoneName`, messages will be like in the follow table.
+If the grabbed item or the drop zone has a `zoneName`, messages will be as per the following table.
 
-| Action                                          | Message template                                                                           |
-|-------------------------------------------------|--------------------------------------------------------------------------------------------|
-| Item was just grabbed                           | `${itemText} grabbed, current position is ${itemPosition} of ${itemsCount} in ${zoneName}` |
-| Item is grabbing over area with another items   | `Grabbing ${itemText}, drop position is ${itemPosition} of ${itemsCount} in ${zoneName}`   |
-| Item is grabbing over area without any items    | `Grabbing ${itemText}, drop in ${zoneName}`                                                |
-| Item is dropped in drop zone with another items | `${itemText} dropped, final position is ${itemPosition} of ${itemsCount} in ${zoneName}`   |
-| Item is dropped in drop zone without any items  | `${itemText} dropped in ${zoneName}`                                                       |
+| Action                                               | Message template                                                                           |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| User grabbed the object                              | `{itemText} grabbed, current position is {itemPosition} of {itemsCount} in {zoneName}`     |
+| Object is over an empty drop zone                    | `Grabbing {itemText}, drop in {zoneName}`                                                  |
+| Object is over a drop zone with other objects        | `Grabbing {itemText}, drop position is {itemPosition} of {itemsCount} in {zoneName}`       |
+| Object is dropped in an empty drop zone              | `{itemText} dropped in {zoneName}`                                                         |
+| Object is dropped in a drop zone with other objects  | `{itemText} dropped, final position is {itemPosition} of {itemsCount} in {zoneName}`       |
 
+## Considerations for developers
```

Comment:
Let's also add "designers" here, since they are usually should think about aria-labels or at least know about the fact that they need to 😅

Reply:
done